plugins/pulp_rpm/plugins/importers/yum/parse/rpm.py

```diff
@@ -127,7 +129,7 @@ def package_signature(headers):
     :param headers: package header
     :type  headers: rpm.hdr
 
-    :return: package signature
+    :return: key id the package was signed with
```

Comment:
More specifically, it's the 8 character abbreviated fingerprint for the key used to sign the package